Resources/views/parts/languages_switcher.html.twig

```diff
@@ -4,19 +4,29 @@
 #    - 'currentLanguage'
 #    - 'availableLanguages'
 #}
-<div id="lang-selector">
-    {# Looping over all available languages to display the links #}
 
-    {% for lang in availableLanguages %}
-        {# This time, we alter the "siteaccess" parameter directly. #}
-        {# We get the right siteaccess with the help of ezpublish.translationSiteAccess() helper #}
+{% if availableLanguages|length > 1 %}
+    <div id="lang-selector" class="transition-showed">
```

Comment:
When the class ``transition-showed`` is used? BTW. the class name is incorrect.
Do you really need ID there?